src/uneditableTextField/UneditableTextField.style.tsx

```diff
@@ -1,9 +1,21 @@
+import React from 'react'
+import cc from 'classcat'
 import styled from 'styled-components'
 
 import { color, font, radius, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
 
 const inputHeight = '54px'
 
+export const StyledUneditableContainer = styled(({ componentType, className, ...props }) =>
+  React.createElement(componentType, {
```

Comment:
I don't thing we use the `componentType` anywhere, so you might be able to simplify this a lot

Reply:
Indeed, I removed the `href` prop it's only used in twice in Kairos.